### DISCUSSION TITLE
Response mismatch when condition fails

### DISCUSSION THREAD

**Discussion by ghost6636** [nathanhoad/godot_dialogue_manager#450] [Q&A] [answered]
In Setting/runtime, Include responses with failed conditions.

Run the following dialogue:
```
- Apple [if false]
	Apple picked.
- Banana
	Banana picked.
```

The _Apple_ option greyed out as expected, but it still says _Apple picked_ when I chose Banana.
The dialogue from that failed response is not skipped, or do I need to tweak something else.

**Comment by nathanhoad** [comment, marked as answer]
Good catch, [v2.32.3](https://github.com/nathanhoad/godot_dialogue_manager/releases/tag/v2.32.3) fixes this issue.
